tests/openvino/test_modeling.py

```diff
@@ -617,8 +622,11 @@ def test_compare_to_transformers(self, model_arch):
         if model_arch == "qwen":
             return
 
-        if model_arch != "chatglm":
+        if not model_arch in ["chatglm", "persimmon"]:
```

Comment:
to fix the style check :
```suggestion
        if model_arch not in ["chatglm", "persimmon"]:
```